catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/integration/test/CatalogPaimonBaseIT.java

```diff
@@ -882,8 +887,30 @@ private void createCatalog() {
     Preconditions.checkArgument(
         StringUtils.isNotBlank(type), "Paimon Catalog backend type can not be null or empty.");
     catalogProperties.put(PaimonCatalogPropertiesMetadata.PAIMON_METASTORE, type);
+
+    // Why needs this conversion? Because PaimonCatalogOperations#initialize will try to convert
+    // Gravitino general S3 properties to Paimon specific S3 properties.
+    Map<String, String> copy = Maps.newHashMap(catalogProperties);
```

Comment:
could you extract the transform logic to a common function, like `IcebergPropertiesUtils.java#toIcebergCatalogProperties`

Reply:
OK, let me take a try.

Reply:
done